Dockerfile

```diff
@@ -52,20 +43,12 @@ COPY modules/material/themes/material/profilereview/* $SSP_PATH/modules/profiler
 COPY modules/material/themes/material/silauth/* $SSP_PATH/modules/silauth/templates/
 
 # Copy in SSP override files
-RUN mv $SSP_PATH/public/index.php $SSP_PATH/public/ssp-index.php
-COPY dockerbuild/ssp-overrides/index.php $SSP_PATH/public/index.php
 COPY dockerbuild/ssp-overrides/saml20-idp-remote.php $SSP_PATH/metadata/saml20-idp-remote.php
 COPY dockerbuild/ssp-overrides/saml20-sp-remote.php $SSP_PATH/metadata/saml20-sp-remote.php
 COPY dockerbuild/config/* $SSP_PATH/config/
-COPY dockerbuild/ssp-overrides/id.php $SSP_PATH/public/id.php
-COPY dockerbuild/ssp-overrides/announcement.php $SSP_PATH/announcement/announcement.php
 COPY dockerbuild/ssp-overrides/sp-php.patch sp-php.patch
 RUN patch /data/vendor/simplesamlphp/simplesamlphp/modules/saml/src/Auth/Source/SP.php sp-php.patch
 
-COPY tests /data/tests
-
-RUN chmod a+x /data/run.sh /data/run-tests.sh
-
 ADD https://github.com/silinternational/config-shim/releases/latest/download/config-shim.gz config-shim.gz
```

Comment:
It might be worth providing a non-gziped version of the config-shim so we can do something like:

```ADD --chmod=755 https://github.com/silinternational/config-shim/releases/latest/download/config-shim /usr/local/bin/config-shim```

See: https://docs.docker.com/reference/dockerfile/#add---chown---chmod

Then actual questions:

- What gets updated more - the above lines or this? (Meaning, should this ADD be higher up, so it doesn't need to be rewritten with as much frequency) I think this would be updated less, but both are probably fairly infrequent.
- Do we need to set this to a specific version (instead of `latest` so Docker will see it needs to updated if we update it?

Reply:
> ...non-gziped version...
Since Go binaries are not small, this could have negative ramifications. I think the extra line in Dockerfile is worth it.

> What gets updated more...
Hard to say. I didn't give much consideration to this, though I agree it's a good way to optimize.

> ...specific version...
Probably so.